schema/spacewalk/upgrade/susemanager-schema-4.4.8-to-susemanager-schema-4.4.9/100-gpg-amazonlinux2023.sql

```diff
@@ -0,0 +1,7 @@
+-- Amazon Linux 2023
+insert into rhnPackageKey (id, key_id, key_type_id, provider_id)
```

Comment:
This insert should also go into `schema/spacewalk/common/data/rhnPackageKey.sql`, otherwise it will be missing when we create the database from scratch.

Reply:
Done, but I will leave a question at that file about it.